x12-edi/2.5/modules/ROOT/pages/index.adoc

```diff
@@ -89,7 +90,7 @@ Individual transaction sets have their own maps, with the following keys:
 |===
 |Key |Description
 |Detail |Map of segments or loops from the detail section of the transaction set. Values are maps for segments or loops that occur once, lists of maps for values that are repeated.
-|Errors (read only) |A list of errors that are associated with the transaction set
+|Errors (read only) |A list of errors that are associated with the transaction set.
 |Group |Map of GS group header segment values. When reading a message, this map is the actual data from the enclosing group, that is, a single map linked from all transaction sets in the group. When writing a message, these values are used for creating the enclosing group. Values not populated in this map then default to the values from the Group map at the message level.
```

Comment:
Fixed for all. GS stands for functional group. It is part of the structure of an X12 envelope. Our users do understand what a GS functional group is because it is key to using X12.